guides/doc-Configuring_Load_Balancer/master.adoc

```diff
@@ -34,16 +34,26 @@ include::common/modules/proc_configuring-remaining-smart-proxy-servers-with-defa
 
 include::common/modules/con_configuring-smart-proxy-server-with-custom-ssl-certificates-for-load-balancing-without-puppet.adoc[leveloffset=+2]
 
-include::common/modules/proc_creating-custom-ssl-certificates-for-smart-proxy-server.adoc[leveloffset=+3]
+:parent-ProductName: {ProductName}
```

Comment:
I looks like "ProductName" is not set in https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes.adoc?plain=1 and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc which means that you cannot/do not have to safe the original value in "parent-productname".

So maybe you can try to just set and unset it without using the "parent-X" and "X: parent" lines.

Reply:
Yep! That did the trick!